pkg/cluster/eks/defaults.go

```diff
@@ -7,8 +7,8 @@ const (
 
 // DefaultImages in each supported location in EC2 (from https://docs.aws.amazon.com/eks/latest/userguide/launch-workers.html)
 var DefaultImages = map[string]string{
-	UsEast1: "ami-0fef2bff3c2e2da93",
-	UsWest2: "ami-0ea01e1d1dea65b5c",
+	UsEast1: "ami-0b2ae3c6bda8b5c06",
+	UsWest2: "ami-08cab282f9979fc7a",
```

Comment:
Please update these in postman collection and OpenAPI

Reply:
I'm not sure we had to update example Ami id's in OpenAPi & Postman, since it's only an example and there are different AMI for different zones, plus it will be more complicated in case of GPU instrances. I think we should make a description elsewhere about correct AMI's.

Reply:
Then remove the example and update the field's description such as point the user to the documentation where the AMIs are listed: https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html

Reply:
Since we update this anyway, can we add the `new` [Irelend])https://aws.amazon.com/about-aws/whats-new/2018/09/amazon-eks-available-in-ireland/) region as well.

Reply:
eu-west-1 image has been added